src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/MainErrorReportViewControllers/TopErrorReportViewModel.swift

```diff
@@ -52,7 +52,7 @@ class TopErrorReportViewModel {
 		if isHistorySectionIncluded {
 			dynamic.add(
 				.section(
-					separators: .all,
+					header: .separator(color: .enaColor(for: .hairline), insets: .init(top: 0, left: 16, bottom: 0, right: 16)),
```

Comment:
Please wrap `.separator` arguments in new lines. Thx ;)

```suggestion
					header: .separator(
						color: .enaColor(for: .hairline),
						insets: .init(top: 0, left: 16, bottom: 0, right: 16)
					),
```